docs/source/getting_started/getting_started.ipynb

```diff
@@ -45,19 +46,151 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Code repository access\n",
+    "### Logging in\n",
     "\n",
-    "After creating your MAAP account, you can create a code repository by navigating to the MAAP GitLab account at https://repo.ops.maap-project.org. This GitLab account is connected to your ADE workspaces automatically when signing into the ADE."
+    "1. Navigate to https://ade.ops.maap-project.org/ in Chrome or Firefox. You should be redirected to a page that looks like this:\n",
+    "![MAAP Project log in](_static/loggingin1.png)\n",
+    "\n",
+    "\n",
+    "2. Click the “Login with EarthData Account” button. If this is your first time logging in, you should be redirected to an EarthData Login page that looks like this:\n",
+    "![Login with EarthData](_static/loggingin2.png)\n",
+    "\n",
+    "\n",
+    "3. Enter your “EarthData Login” account credentials here and click “Log in”. You should see a temporary page that says “Redirecting”, followed by the MAAP showing your Workspaces (which will be empty to start):\n",
```

Comment:
I always get redirected back to 
![Screenshot 2023-05-02 at 7 57 41 AM](https://user-images.githubusercontent.com/114033465/235704738-91d5e6a8-5f84-4272-8e2a-d1cf5f8b8658.png)
Where I need to do tools -> development

Reply:
But Rob said: weird. it brings me to the workspace / eclipse-che dashboard

Curious to see what other people see